src/data/api.js

```diff
@@ -52,3 +53,25 @@ export async function submitAttempt(attemptId) {
 export async function endExamWithFailure(attemptId, error) {
   return updateAttemptStatus(attemptId, ExamAction.ERROR, error);
 }
+
+export async function fetchProctoringSettings() {
+  return {
+    proctorint_settings: {
+      link_urls: {
+        contact_us: 'https://example.com/contact_us/',
+        faq: 'https://example.com/faq/',
+        online_proctoring_rules: 'https://example.com/online_proctoring_rules/',
+        tech_requirements: 'https://example.com/tech_requirements/',
+      },
+    },
+    proctoring_backends: {
+      software_secure: {
+        exam_register_endpoint: '{add endpoint to SoftwareSecure}',
```

Comment:
Here and below:
SoftwareSecure was the first and only one proctored provider supported by edX, now we have much more, let's change it s/SoftwareSecure/Proctored Provider/

Reply:
It will be fixed in that PR https://github.com/raccoongang/frontend-lib-special-exams/pull/12

Reply:
Yep, I've already found an answer to my question.